src/utils/BulbBotUtils.ts

```diff
@@ -119,7 +119,9 @@ export default class {
 		const features: string[] = [];
 
 		guildFeatures.forEach((feature) => {
-			features.push(`${Emotes.features[feature]} [\`${feature}\`](https://bulbbot.rocks '${GuildFeaturesDescriptions[feature]}')`);
+			const fEmoji = Emotes.features[feature] ? Emotes.features[feature] : Emotes.features.Feature;
```

Comment:
```suggestion
			const fEmoji = Emotes.features[feature] ?? Emotes.features.Feature;
```
A better solution would be this, `??` is a [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).

Also you could change the name `Feature` to `DEFAULT` or something like that, makes it more readable!

Reply:
👍 will update when I get home